src/Native/System.Net.Http.Native/pal_easy.cpp

```diff
@@ -0,0 +1,113 @@
+static_assert(PAL_CURLOPT_ACCEPT_ENCODING == CURLOPT_ACCEPT_ENCODING, "");
+static_assert(PAL_CURLOPT_PRIVATE == CURLOPT_PRIVATE, "");
+static_assert(PAL_CURLOPT_COPYPOSTFIELDS == CURLOPT_COPYPOSTFIELDS, "");
+static_assert(PAL_CURLOPT_SEEKDATA == CURLOPT_SEEKDATA, "");
+static_assert(PAL_CURLOPT_USERNAME == CURLOPT_USERNAME, "");
+static_assert(PAL_CURLOPT_PASSWORD == CURLOPT_PASSWORD, "");
+
+static_assert(PAL_CURLOPT_WRITEFUNCTION == CURLOPT_WRITEFUNCTION, "");
+static_assert(PAL_CURLOPT_READFUNCTION == CURLOPT_READFUNCTION, "");
+static_assert(PAL_CURLOPT_HEADERFUNCTION == CURLOPT_HEADERFUNCTION, "");
+static_assert(PAL_CURLOPT_SSL_CTX_FUNCTION == CURLOPT_SSL_CTX_FUNCTION, "");
+static_assert(PAL_CURLOPT_SEEKFUNCTION == CURLOPT_SEEKFUNCTION, "");
+
+static_assert(PAL_CURLE_OK == CURLE_OK, "");
+static_assert(PAL_CURLE_UNSUPPORTED_PROTOCOL == CURLE_UNSUPPORTED_PROTOCOL, "");
+static_assert(PAL_CURLE_NOT_BUILT_IN == CURLE_NOT_BUILT_IN, "");
+static_assert(PAL_CURLE_COULDNT_RESOLVE_HOST == CURLE_COULDNT_RESOLVE_HOST, "");
+
+static_assert(PAL_CURLINFO_PRIVATE == CURLINFO_PRIVATE, "");
+static_assert(PAL_CURLINFO_HTTPAUTH_AVAIL == CURLINFO_HTTPAUTH_AVAIL, "");
+
+static_assert(PAL_CURLAUTH_None == CURLAUTH_NONE, "");
+static_assert(PAL_CURLAUTH_Basic == CURLAUTH_BASIC, "");
+static_assert(PAL_CURLAUTH_Digest == CURLAUTH_DIGEST, "");
+static_assert(PAL_CURLAUTH_Negotiate == CURLAUTH_GSSNEGOTIATE, "");
+
+static_assert(PAL_CURLPROXY_HTTP == CURLPROXY_HTTP, "");
+
+static_assert(PAL_CURLPROTO_HTTP == CURLPROTO_HTTP, "");
+static_assert(PAL_CURLPROTO_HTTPS == CURLPROTO_HTTPS, "");
+
+extern "C" CURL* EasyCreate()
+{
+    return curl_easy_init();
+}
+
+extern "C" void EasyDestroy(CURL* handle)
+{
+    curl_easy_cleanup(handle);
+}
+
+#define convertOption(option) static_cast<CURLoption>(option)
+
+extern "C" int32_t EasySetOptionString(CURL* handle, PAL_CURLoption option, const char* value)
+{
+    return curl_easy_setopt(handle, convertOption(option), value);
+}
+
+extern "C" int32_t EasySetOptionLong(CURL* handle, PAL_CURLoption option, int64_t value)
+{
+    return curl_easy_setopt(handle, convertOption(option), value);
+}
+
+extern "C" int32_t EasySetOptionPointer(CURL* handle, PAL_CURLoption option, void* value)
+{
+    return curl_easy_setopt(handle, convertOption(option), value);
+}
+
+extern "C" const char* EasyGetErrorString(PAL_CURLcode code)
+{
+    return curl_easy_strerror(static_cast<CURLcode>(code));
+}
+
+#define convertInfo(info) static_cast<CURLINFO>(info)
+
+extern "C" int32_t EasyGetInfoPointer(CURL* handle, PAL_CURLINFO info, void** value)
+{
+    return curl_easy_getinfo(handle, convertInfo(info), value);
+}
+
+extern "C" int32_t EasyGetInfoLong(CURL* handle, PAL_CURLINFO info, int64_t* value)
+{
+    return curl_easy_getinfo(handle, convertInfo(info), value);
+}
+
+extern "C" int32_t EasyPerform(CURL* handle)
+{
+    return curl_easy_perform(handle);
+}
+
+extern "C" int32_t EasyPause(CURL* handle)
```

Comment:
This function is now named strangely.  It sounds like it's pausing, but because it's hard coded to use CURLPAUSE_CONT, it's actually unpausing.